srcpkgs/direwolf/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'direwolf'
+pkgname=direwolf
+version=1.6
+revision=1
+build_style=cmake
+hostmakedepends="gcc make hamlib"
+makedepends="alsa-lib-devel libudev-devel hamlib-devel"
+depends="hamlib"
```

Comment:
```
=> direwolf-1.6_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libhamlib.so.2 <-> hamlib>=3.3_1
   SONAME: libasound.so.2 <-> alsa-lib>=1.0.20_1
   SONAME: libudev.so.1 <-> eudev-libudev>=1.6_1
   SONAME: libc.so <-> musl>=1.1.21_1
```

The `depends` line isn't necessary because it's detected automatically by our build system.